Firmware/Marlin_main.cpp

```diff
@@ -750,23 +750,32 @@ static void factory_reset(char level)
 	case 2: // Level 2: Prepare for shipping
 		factory_reset_stats();
 		// [[fallthrough]] // there is no break intentionally
+	case 4: // Level 4: Preparation after being serviced
+		// Force language selection at the next boot up.
+		lang_reset();
+		// Force the "Follow calibration flow" message at the next boot up.
+		calibration_status_store(CALIBRATION_STATUS_Z_CALIBRATION);
+		eeprom_write_byte((uint8_t*)EEPROM_WIZARD_ACTIVE, 2); //run wizard
+		farm_mode = false;
+		eeprom_update_byte((uint8_t*)EEPROM_FARM_MODE, farm_mode);
 
-		case 3:
-			menu_progressbar_init(EEPROM_TOP, PSTR("ERASING all data"));
-			Sound_MakeCustom(100,0,false);
-			
-			// Erase EEPROM
-			for (uint16_t i = 0; i < EEPROM_TOP; i++) {
-				eeprom_update_byte((uint8_t*)i, 0xFF);
-				menu_progressbar_update(i);
-			}
-			menu_progressbar_finish();
-			softReset();
-
+#ifdef FILAMENT_SENSOR
+		fsensor_enable();
+		fsensor_autoload_set(true);
+#endif //FILAMENT_SENSOR
+		break;
+	case 3:
+		menu_progressbar_init(EEPROM_TOP, PSTR("ERASING all data"));
+		Sound_MakeCustom(100,0,false);
```

Comment:
the sound call has been extracted to the beginning of this function, please remove this `Sound_MakeCustom(100,0,false);` line from here

Reply:
If you need it right now for testing, please remove it.